Service/Model.Implementation.Test/CategoryControllerTest.cs

```diff
@@ -0,0 +1,278 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+
+using NUnit.Framework;
+using Rhino.Mocks;
+
+using TodoSystem.Service.Model.Interface;
+
+namespace TodoSystem.Model.Implementation.Test
+{
+    [TestFixture]
+    public class CategoryControllerTest
+    {
+        /// <summary>
+        /// Storage implementation for <see cref="ICategoryController"/>
+        /// </summary>
+        private sealed class CategoryController : BaseCategoryController
```

Comment:
If you just leave one CategoryController with out base class you do not need this strange fake class